src/rgw/rgw_admin.cc

```diff
@@ -4817,7 +4817,11 @@ int main(int argc, char **argv)
     string user_str = user_id.to_str();
     int ret = store->cls_user_get_header(user_str, &header);
     if (ret < 0) {
-      cerr << "ERROR: can't read user header: " << cpp_strerror(-ret) << std::endl;
+      if (ret == -ENOENT) { /* in case of ENOENT */
+        cerr << "User has not been initialized or user does not exist" << std::endl;
+      } else {
+        cerr << "ERROR: can't read user: " << cpp_strerror(-ret) << std::endl;
```

Comment:
for posterity, cpp_strerror coerces whatever code is passed to be positive, so there's no need to make a code negative